integration/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "Integration testing for Amazon Chime SDK React Components Library",
+  "scripts": {
+    "test": "npm ci && ./script/run-test",
```

Comment:
@xuesichao I was trying to run the demo and when I run `npm run test` in `integration` folder I am getting this error: 
```
Starting the test demo
Waiting for test demo to start

> test-demo@0.1.0 start
> concurrently "npm run start:client" "npm run start:backend"

sh: concurrently: command not found
```

Seems like since concurrently is not a dependency in integration folder's `package.json` the test command is failing. You might want to add a `npm install` in `run-test` to resolve this.

Reply:
Please refer to the `package.json` in JS SDK as that is calling `npm run start` in test demo and test demo's start is installing deps here: https://github.com/aws/amazon-chime-sdk-js/blob/master/integration/js/app/package.json#L10

Reply:
There are two options either: `npm install` in `run-test` or updated `package.json`

Reply:
Thanks for catching this! I added `npm install` in the run-test script, after `cd integration/app/test-demo`